values.yaml

```
@@ -280,6 +280,9 @@ webserver:
   # Number of webservers
   replicas: 1
 
+  # Prevent airflow webserver reboot only for Airflow 2.0.0
+  useDefaultAirflowImage: true
```

```suggestion
  # Prevent airflow webserver reboot only for Airflow 2.0.0
  useDefaultAirflowImage: false
```

Setting this to `false` by default until we have 2.0 as the default image.